src/utils/sorted-bins.js

```diff
@@ -18,6 +18,7 @@ export default class SortedBins {
   /**
    * Get an array of object with sorted count and index of bins
    * @param {Number} lower
```

Comment:
Nit: This doesn't correctly document the params (looks like `f(lower, upper)`) . But probably no need to change this now. We'll probably move to something like flow type annotation comments in the future. A more correct version of JSDoc would probably look like:
```
@param {Number[]} range
@param {Number} range[0] - lower bound
...
```